src/python/utils/index_partition.py

```diff
@@ -35,6 +35,23 @@ def deserialize(self, in_file):
             self._partition = pickle.load(payload)
             payload.close()
 
+    def get_token_list(self):
+        return self._partition.get_token_list()
+
+    def get_token_count(self,key,version):
+        if(key not in self._partition._partition.keys()):
+            return -1
+        if(self._partition._partition[key]['ngram_size']!=1):
+            return -1
+
+        count = 0
+        for doc in self._partition._partition[key]['documentOccurrences']:
+            for v in doc['versions']:
```

Comment:
As commented above, just get the information for the latest version of each document